Patch-Source: https://gitlab.archlinux.org/archlinux/packaging/packages/python-prctl/-/blob/42e41b4b69d20ff1d76d82fa1c3f4e26c29ac3c7/setuptools-62.1.diff

diff --git a/test_prctl.py b/test_prctl.py
index df84986..66e5c4f 100644
--- a/test_prctl.py
+++ b/test_prctl.py
@@ -20,7 +20,7 @@ except ImportError:
     pass
 
 curdir = os.path.dirname(__file__)
-builddir = os.path.join(curdir, 'build', 'lib.%s-%s' % (distutils.util.get_platform(), sys.version[0:3]))
+builddir = os.path.join(curdir, 'build', 'lib.%s-%s' % (distutils.util.get_platform(), sys.implementation.cache_tag))
 
 # Always run from the builddir
 if not os.path.exists(builddir) or \
